### PR TITLE
ELEMENTS-1677: build failed while generating a snapshot for elements project

### PR DESCRIPTION
**Pull request by rahuljain-dev**
https://jira.nuxeo.com/browse/ELEMENTS-1677
(backward port)

Issue raised in lerna repo:
https://github.com/lerna/lerna/issues/3854

### CODE REVIEW

**Review by rakeshkumar1019 — DISMISSED** [review]
Looks good.Just add the PR raised in lerna repo of this issue for future reference.

**Review by poonamyadav252 — COMMENTED** [review]
I am not able to see story books checks here.

**Comment by rahuljain-dev** [comment]
> Looks good.Just add the PR raised in lerna repo of this issue for future reference.
added

File: .github/workflows/main.yaml [hunk 1]
```diff
@@ -1,5 +1,10 @@
 name: Build
 
+## limiting lerna version as latest is not compatible with node 14.
+## We will revert this once node is upragded.
+env:
+  lerna: 'lerna@6.6.2'
+
```
Comment (poonamyadav252) [inline .github/workflows/main.yaml]: Variable name should be in upper case.

**Comment by rahuljain-dev** [comment]
> I am not able to see story books checks here.

which story book checks?

**Comment by poonamyadav252** [comment]
> > I am not able to see story books checks here.
> 
> which story book checks?

https://github.com/nuxeo/nuxeo-elements/actions/runs/6391071771/job/17345592528?pr=813

File: .github/workflows/main.yaml (hunk 1 above)
Comment (rahuljain-dev) [reply .github/workflows/main.yaml]: Updated

**Pull request closed without merging**